eng/common/scripts/Invoke-DevOpsAPI.ps1

```diff
@@ -158,5 +158,26 @@ function Add-RetentionLease {
           -Headers (Get-DevOpsApiHeaders -Base64EncodedToken $Base64EncodedAuthToken) `
           -MaximumRetryCount 3 `
           -ContentType "application/json"
+}
+
+function Get-PackagesInArtifactFeed {
+  param (
+    $Organization="azure-sdk",
+    $Project="public",
+    [Parameter(Mandatory = $true)]
+    $FeedId,
+    $ApiVersion="6.1-preview.1",
```

Comment:
Why we use preview version by default?

Reply:
No particular reason. It apears to be the latest release.